File: katas/8 kyu/convert-a-boolean-to-a-string/solution.py
# convert-a-boolean-to-a-string
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
boolean_to_string = str
# ----------------------------------- TEST ----------------------------------- #
@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(boolean_to_string(True), "True")
        test.assert_equals(boolean_to_string(False), "False")

File: katas/8 kyu/convert-a-boolean-to-a-string/solution.js
// convert-a-boolean-to-a-string
/* ------------------------------- MY SOLUTION ------------------------------ */
function booleanToString(b){
  return String(b)
}
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
function booleanToString(b){
  return b.toString();
}
/* ---------------------------------- TEST ---------------------------------- */
const chai = require("chai");
const Test = chai.assert.strictEqual;
chai.config.truncateThreshold=0;


describe("Tests", () => {
  it("test", () => {
Test(booleanToString(true), "true", 'When we pass in true, we want the string "true" as output');
Test(booleanToString(false), "false", 'When we pass in false, we want the string "false" as output');
  });
});
